README.md

```diff
@@ -61,13 +34,13 @@ For more information on the importance of a professional README for your reposit
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+
+# 📖 [eHllo microverse!] <a name="Hello microverse!"></a>
 
 > Describe your project in 1 or 2 sentences.
+the project has h1 hello microverse with green color style 
```

Comment:
- [ Required ] I think it would be better if you remove the instructional text from your readme file, just like to logo the instructional text is supposed to guide you on how to fill the different parts of the readme. Once these sections have been filled you are supposed to delete them. Please see the screenshot below to see which sections to delete.

- [ Required ] I think it would be better if you check on the `project title` in your readme file. I think it should be `Hello Microverse!` and not `ehllo Microverse!`

## Screenshot
![Screenshot from 2023-07-07 00-29-00](https://github.com/abenisamuel11/hello-microverse-community1/assets/16392046/2e25d6d8-588c-4c9f-9ee9-ea010428bf3d)